### DISCUSSION TITLE
Video DownloadHelper does not work with Hotaudio.net

### DISCUSSION THREAD

**Discussion by Gurren828** [aclap-dev/video-downloadhelper#1420] [Help and Q&A]
### Your question

Are there any plans to be able to use VDH on hotaudio.net?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"5983fdc6b3894c12"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\ruben\\Music\\iTunes\\iTunes Media\\Music\\L0LABITES",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\ruben\\Music\\iTunes\\iTunes Media\\Music\\RoteDisaster"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 56,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by Mattmjs** [comment]
I don't think it will work on this site, it does not use a simple mp3 format for example. They use what looks like an unorthodox method
with a .hax extension which is labelled as an octtet stream. Just use a something to record the audio as it's sent through your computer.
